docs/docs/nlu-only.mdx

```diff
@@ -51,3 +51,31 @@ To do this, run:
 ```bash
 curl localhost:5005/model/parse -d '{"text":"hello"}'
 ```
+
+## Connecting to an NLU server
+
+
+You can connect a Rasa dialogue management only server with a separately running NLU server
+by adding the connection details to the dialogue management server's endpoint configuration file:
+
+```yaml-rasa title="endpoints.yml"
+nlu:
+    url: "http://<your nlu host>:<your nlu port>"
+    token: <token>  # [optional]
+    token_name: <name of the token> # [optional] (default: token)
+```
+
+The `token` and `token_name` refer to optional [authentication parameters](./http-api.mdx#token-based-auth).
+
+The dialogue management server should serve a model that does not include an NLU model.
+To obtain a dialogue management only model, train a model with `rasa train core` or use 
+`rasa train` but exclude all NLU data.
```

Comment:
I would put this part at the top where we introduce the "dialogue management only server", or create another short header explaining what that is and how to spin one up